omnigibson/transition_rules.py

```diff
@@ -638,8 +637,8 @@ def transition(self, individual_objects, group_objects):
     DicingRule(),
     # Strawberry smoothie
     BlenderRule(
-        output_system=StrawberrySmoothieSystem,
-        particle_requirements={MilkSystem: 10},
+        output_system=get_system("strawberry_smoothie"),
+        particle_requirements={get_system("milk"): 10},
```

Comment:
meta question: should these top-level APIs take in the system string names or the systems themselves? it seems a bit bulky to always pass in `get_system(...)`, but perhaps this makes it less bug-prone and also keeps the same structure as our main object state hierarchy (which also maps classes rather than strings). what do you think?

Reply:
I think they should take in the system string names, to be consistent with the category argument in `DatasetObject(category="novel_category", ...)`. This assumes we will have infinite number of systems and we won't be able to enumerate them.

Reply:
Make output_system take in `"strawberry_smoothie"` rather than `get_system("strawberry_smoothie")`

Reply:
done